WeRadiate-ThermoSense.ino

```diff
@@ -405,6 +464,214 @@ void loop(void)
         {
         // put your main code here, to run repeatedly:
         gCatena.poll();
-        Serial.println("hello world");
-        delay(60000);
+        }
+
+void startSendingUplink(void)
```

Comment:
See my more recent routines. This needs to change to 
1) do measurements (building buffer)
2) transmit results.

Reply:
I looked at 461x_test01 and I am not sure where to change. Could you be more specific?